README.md

```diff
@@ -59,14 +59,14 @@ If you have a Git credential helper configured, you should not be prompted for y
 
 #### Option #1 (Recommended) -  Project-centric approach
 
-This option will output all the project binaries in the project's build folder e.g. c:/path/to/o3de-multiplayersample/build
+This option will output all the project binaries in the project's build folder e.g. c:/path/to/o3de-multiplayersample/build/windows_vs2019 using Visual Studio 2019. Refer to the [project configuration](https://www.o3de.org/docs/user-guide/build/configure-and-build/#configuring-projects) documentation for more details and options.
 
 ```shell
 # example configure command
-> cmake -S c:/path/to/o3de-multiplayersample -B c:/path/to/o3de-multiplayersample/build/windows_vs2019 -G "Visual Studio 16" -DLY_3RDPARTY_PATH="c:/3rdparty"
+> cmake --build build/windows_vs2019 -G "Visual Studio 16" -DLY_3RDPARTY_PATH="c:/3rdparty"
```

Comment:
I think we should remove -DLY_3RDPARTY_PATH="c:/3rdparty". A default setting is good enough. 

> cmake --build build/windows_vs2019 -G "Visual Studio 16"

Reply:
In fact, I dont think we need to specify VS2019, since VS2022 works quite well now esp in 22.10.

Reply:
Would argue that its very useful to explicitly set the 3rd party path as you quickly tear through disc space if you don't control cache.